decidim-initiatives/app/views/decidim/initiatives/create_initiative/_committee_member.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="flex items-center justify-between my-1">
+  <%= cell "decidim/author", ::Decidim::UserPresenter.new(request.user), layout: :compact %>
```

Comment:
```suggestion
  <%= cell "decidim/author", request.user.presenter, layout: :compact %>
```

Reply:
is not very clear for me if here needs to be current_user or it needs to be the user / group that created the initiative. 

If you set the initiative's author to user_group, in the InitiativesCommitteeMember you get the current user.

Reply:
here the name `request` is misleading but I followed the naming since it is used in other places. Request is the model representing the request (or invitation) to join the initiative as a member. The view shows a list of members that have been invited (or have requested) to the initiative, so, we pass the associated `request.user` to the cell.